From e0d2caafcd5e583a01f7792eaec5d1f2d540bdf1 Mon Sep 17 00:00:00 2001
From: Rudi Heitbaum <rudi@heitbaum.com>
Date: Fri, 18 Nov 2022 22:44:52 +1100
Subject: [PATCH] [Media Common] [VP] dont include va_dricommon.h

<va/va_dricommon.h> is not in non-X11 libva 2.16.0
The header "media_libva_putsurface_linux.h" is included

Downloaded from upstream commit
https://github.com/intel/media-driver/commit/e0d2caafcd5e583a01f7792eaec5d1f2d540bdf1

Signed-off-by: Rudi Heitbaum <rudi@heitbaum.com>
Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---
 media_softlet/linux/common/vp/ddi/ddi_vp_functions.cpp | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/media_softlet/linux/common/vp/ddi/ddi_vp_functions.cpp b/media_softlet/linux/common/vp/ddi/ddi_vp_functions.cpp
index 8695d6c86..6719c07b3 100644
--- a/media_softlet/linux/common/vp/ddi/ddi_vp_functions.cpp
+++ b/media_softlet/linux/common/vp/ddi/ddi_vp_functions.cpp
@@ -23,7 +23,6 @@
 //! \file     ddi_vp_functions.cpp
 //! \brief    ddi vp functions implementaion.
 //!
-#include <va/va_dricommon.h>
 #include "ddi_vp_functions.h"
 #include "media_libva_util_next.h"
 #include "media_libva_common_next.h"
@@ -4700,4 +4699,4 @@ VAStatus DdiVpFunctions::PutSurfaceLinuxHW(
 
     return VA_STATUS_SUCCESS;
 }
-#endif // !defined(ANDROID) && defined(X11_FOUND)
\ No newline at end of file
+#endif // !defined(ANDROID) && defined(X11_FOUND)
